spec/src/modules/tracker.js

```diff
@@ -615,14 +615,14 @@ describe('ConstructorIO - Tracker', () => {
         fetch: fetchSpy,
       });
 
-      expect(tracker.trackPurchase(parameters)).to.equal(true);
+      expect(tracker.trackPurchase({})).to.equal(true);
```

Comment:
We were passing `section` as part of the parameters, which defeats the point of the test as we're testing to see that `section` is defaulted when not supplied.